fighthealthinsurance/ml/ml_models.py

```
@@ -2271,4 +2271,76 @@ def models(cls) -> List[ModelDescription]:
+    def external(self):
+        return False
+
+    @classmethod
+    def _resolve_tailscale_host(cls, hostname: str) -> Optional[str]:
+        """Try to resolve a Tailscale hostname via DNS."""
+        import socket
+
+        try:
+            # Try to resolve the hostname
+            socket.gethostbyname(hostname)
+            return hostname
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)